src/pages/sylvia/basic/contract-structure.md

```diff
@@ -0,0 +1,107 @@
+# Contract structure
+
+Sylvia contracts are designed using the actor model. The actor is a contract
+`struct`, which can store a state, and on which we implement a behavior.
+
+```rust
+use sylvia::cw_storage_plus::Item;
+
+pub struct CounterContract {
+    pub count: Item<u64>,
+}
+```
+
+Item is an accessor that allows to read a state stored on the blockchain via the
+key. It doesn't hold any state by itself.
```

Comment:
I'd avoid explaining `Item` here. I'd mention that we keep the state accessors as part of a contract definition, but instead of explaining it I'd mention those are normal storage plus primitives and link to the proper doc page.